### PR TITLE
Feature/helper viewlist filter

### PR DESCRIPTION
**Pull request by janwidmer**
## Purpose of this pull request? 

* Enhancement

The pull request refers to the:

* nitro app (@nitro/app)

## What changes did you make?

Enhancing the viewlist helper to take filter arguments to include / exclude views.

### CODE REVIEW

File: packages/nitro-app/app/templating/hbs/helpers/viewlist.js [hunk 1]
```diff
@@ -1,17 +1,59 @@
+/**
+ * Viewlist helper, can be used to display lists of views filtered by specific include / exclude terms
+ *
+ * Usage:
+ *
+ * Without parameter, all views will be displayed
+ * {{viewlist}}
+ *
+ * With parameter viewIncludes, all views containing at least one of the terms will be displayed
+ * {{viewlist viewIncludes="<term-1>;<term-2>"}}
```
Comment (ernscht) [inline packages/nitro-app/app/templating/hbs/helpers/viewlist.js]: I would prefer using a simpler property for viewIncludes and viewExcludes
What about include & exlude? 
At the same time, one could consider whether semicolon or comma is better suited as separator

File: packages/nitro-app/app/templating/hbs/helpers/viewlist.js [hunk 2]
```diff
@@ -1,17 +1,59 @@
...
+			});
+		});
+	}
+
+	filteredViews.forEach((viewItem) => {
 		markup.push(`<li><a href="/${viewItem.url}">${viewItem.name}</a></li>`);
 	});
 
+	if (filteredViews.length === 0) {
+		markup.push('<li>For this include / exclude term, no pages where found</li>');
```
Comment (ernscht) [inline packages/nitro-app/app/templating/hbs/helpers/viewlist.js]: typo ;-)

File: packages/nitro-app/app/templating/hbs/helpers/viewlist.js (hunk 1 above)
Comment (janwidmer) [reply packages/nitro-app/app/templating/hbs/helpers/viewlist.js]: i will rename the params. Regarding separator, I have no hard feelings, happy to use ","?

Comment (janwidmer) [reply packages/nitro-app/app/templating/hbs/helpers/viewlist.js]: switched to comma separator

File: packages/nitro-app/app/templating/hbs/helpers/viewlist.js (hunk 2 above)
Comment (janwidmer) [reply packages/nitro-app/app/templating/hbs/helpers/viewlist.js]: Typo fixed

**Pull request merged**